src/resistance.rs

```diff
@@ -0,0 +1,170 @@
+//! Types and constants for handling electrical resistance.
+
+use super::measurement::*;
+
+/// The `Resistance` struct can be used to deal with electrical resistance in a
+/// common way.
+///
+/// # Example
+///
+/// ```
+/// use measurements::Resistance;
+///
+/// let r = Resistance::from_kiloohms(470.0);
+/// let ko = r.as_kiloohms();
+/// let mo = r.as_megaohms();
+/// println!("A 4.7 kΩ resistor has {} Ω or {} MΩ", ko, mo);
```

Comment:
The example is 470 kΩ, so this example text is wrong.

Reply:
whoops, thanks :)